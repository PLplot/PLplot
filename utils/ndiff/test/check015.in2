    |\^/|     Maple V Release 5.1 (University of Utah)
._|\|   |/|_. Copyright (c) 1981-1998 by Waterloo Maple Inc. All rights
 \  MAPLE  /  reserved. Maple and Maple V are registered trademarks of
 <____ ____>  Waterloo Maple Inc.
      |       Type ? for help.
## /u/sy/beebe/jim-ball/toms/quadpack/main2.maple, Wed Jan 26 08:01:37 2000
## Edit by Nelson H. F. Beebe <beebe@math.utah.edu>
> 
## Suppress "bytes used ..." and other status messages
> interface(quiet = true);
### Numerical integration with Maple V5.1
###
### int(sin(x) * x^alpha * exp(-x) * ln(x), x = 0..infinity))
### Digits := 50:
### for alpha from 0 by 0.5 to 100
### do
###     printf("%d\t%.1f\t%d\t%60.50e\n",
### 	1,
### 	alpha,
### 	0,
### 	evalf(int(sin(x) * x^alpha * exp(-x) * ln(x), x = 0..infinity)));
### od;
1       0.0     0          -6.91955458920286e-02
1       .5      0           7.41895901555803e-03
1       1.0     0           3.81053724092472e-02
1       1.5     0           1.61420869583317e-02
1       2.0     0          -1.04593709289476e-01
1       2.5     0          -4.30865150235775e-01
1       3.0     0          -1.17809724509617e+00
1       3.5     0          -2.74966469716760e+00
1       4.0     0          -5.83482672464782e+00
