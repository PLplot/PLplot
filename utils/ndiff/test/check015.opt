--relative-error 1.e-14 --silent
