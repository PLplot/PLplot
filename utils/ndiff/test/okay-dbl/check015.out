### Maximum relative error in matching lines = 8.62e-15 at line 27 field 4
